src/test/scala/com/exasol/cloudetl/bucket/AlluxioBucketTest.scala

```diff
@@ -0,0 +1,21 @@
+package com.exasol.cloudetl.bucket
+
+class AlluxioBucketTest extends AbstractBucketTest {
```

Comment:
Do you have an integration test?

Reply:
Not at the moment. I have searched, they indeed have a docker image. And also test-container integration but not out of shelf (as in localstack).
I am going to add it later (or in another ticket).

Adding [`alluxio-tests`](https://search.maven.org/artifact/org.alluxio/alluxio-tests) dependency should help. 

Reference test:
- https://github.com/Alluxio/alluxio/blob/master/tests/src/test/java/alluxio/master/table/TableIntegrationTest.java